.gitignore

```diff
@@ -14,4 +14,6 @@ results
 npm-debug.log
 node_modules
 .DS_Store
+.idea
 package-lock.json
+.idea
```

Comment:
If you can remove one of these .idea ignores, we are good to go. Thx!

Reply:
ah, sure. Removed.